src/proxy/momento/src/main.rs

```diff
@@ -302,13 +302,16 @@ async fn spawn(
     }
     let auth_token =
         std::env::var("MOMENTO_AUTHENTICATION").expect("MOMENTO_AUTHENTICATION must be set");
-    let credential_provider = CredentialProviderBuilder::from_string(auth_token)
+    let credential_provider = CredentialProvider::from_string(auth_token).unwrap_or_else(|e| {
+        eprintln!("failed to initialize credential provider. error: {e}");
+        std::process::exit(1);
+    });
+    let client_builder = match CacheClient::builder()
+        .default_ttl(DEFAULT_TTL)
+        .configuration(configurations::Laptop::latest())
```

Comment:
What's the reason for choosing the "laptop" configuration here?

Reply:
That is our config that has the most lax timeouts and settings. It's the default in our SDKs as well. We can make this configurable in a followup PR.

Reply:
SGTM.